src/Symfony/Component/Ldap/Entry.php

```diff
@@ -18,11 +18,16 @@ class Entry
 {
     private $dn;
     private $attributes;
+    private $force_lowercase_attributes;
```

Comment:
Variables should be named in camelCase

Reply:
Ok, I'll change it. I'm surprised the PSR tests didn't pick that up.

Reply:
friendly ping